### PR TITLE
Named burgers are back!

### LINKED ISSUE #12012: Human burgers are no longer named after their victim
At least not the table crafted ones. I haven't tried it with the burgers created by clicking a bun with a piece of meat.

Not a bug I know, but it'd be a fun thing to bring back. Not sure why it wasn't transferred over to the new system when it was coded.

### LINKED ISSUE #15664: Using monkeys in the gibber drops human skin
as opposed to monkey skin

### LINKED ISSUE #15706: Issue with augment surgery/health
HELP: PupGaming/(Gases-The-Nerds) (?) (PP) (VV) (SM) (FLW) (TP) (REJT): so I think there is a bug where when you get full aug the client things that the last part auged is still injured whne its not 

http://i.imgur.com/h1qPesj.png

At half health but no damage

### PR DESCRIPTION
**Pull request by Core0verload**
- Human burgers are, once again, named against generous meat donors that made them possible. This means you can finally take a seat, relax and enjoy a "Captain Comdom burger" after a successful revolution. Fixes #12012. 
- Gibber no longer produces human skin while gibbing aliens or monkeys. Fixes #15664.
- Aug. surgery is now updating health correctly. It isn't related to burgers, meat and cooking, but an extra fix wouldn't hurt, right? Fixes  #15706.
- Liposaction surgery now respects species meat types and passes down job and name to created meat.

:cl: CoreOverload
rscadd: Human burgers are, once again, named after the generous meat donors that made them possible.
/:cl:

### CODE REVIEW

**Comment by TheRealGLH** [comment]
Could you also do this for named simple animals, or is that out of scope for this PR?
Would be neat to be able to taunt the HoP or the bartender with named Ian or Pun-Pun burgers.

File: code/modules/food&drinks/kitchen machinery/gibber.dm [hunk 1]
```diff
@@ -183,11 +184,15 @@
...
-	else
-		if(iscarbon(occupant))
-			var/mob/living/carbon/C = occupant
-			typeofmeat = C.type_of_meat
-			gibtype = C.gib_type
+	else if(iscarbon(occupant))
+		var/mob/living/carbon/C = occupant
+		typeofmeat = C.type_of_meat
+		gibtype = C.gib_type
+		if(ismonkey(C))
```
Comment (Supermichael777) [inline code/modules/food&drinks/kitchen machinery/gibber.dm]: quite honestly this needs to be a check like the ones above that produces what the species datum has and a has skin flag.

Comment (Core0verload) [inline code/modules/food&drinks/kitchen machinery/gibber.dm]: Only humans have species datums. The rest of carbons don't even have a typeofskin var.

**Comment by Cheridan** [comment]
never really liked this feature tbh but it sounds like something oldfags would yell about being gone so *shrug

**Pull request merged**